templates/publisher/settings.html

```diff
@@ -1,276 +1,270 @@
-                  {% endfor %}
-                 " />
-                <div class="js-multiselect-holder{% if blacklist_country_codes %} is-disabled{% endif %}"></div>
-              </div>
-              <div class="js-blacklist p-multiselect__wrapper">
-                <input type="radio" name="territories_custom_type" value="blacklist" id="territories_custom_blacklist"
-                  {% if blacklist_country_codes %}
-                       checked="checked"
-                  {% endif %}
-                />
-                <label for="territories_custom_blacklist">Exclude these territories</label>
-                <input type="text" class="js-multiselect-input" name="blacklist_countries" value="
-                  {% for territory in blacklist_country_codes %}
-                    {{ territory }}{{ "," if not loop.last }}
-                  {% endfor %}
-                 " />
-                <div class="js-multiselect-holder{% if whitelist_country_codes or not blacklist_country_codes %} is-disabled{% endif %}"></div>
-              </div>
-            </div>
-          </div>
+          <hr class="u-no-margin--bottom" />
         </div>
+      </div>
 
-        <div class="p-strip is-shallow">
-          <div class="row">
-            <hr class="u-no-margin--bottom" />
+      <div class="row">
+        <div class="col-2">
+          <label>
+            Distribution:
+          </label>
+        </div>
+        <div class="col-8">
+          <label>This snap should be available in:</label>
+          <input type="radio" name="territories" value="all" id="territories_all"
+                 {% if not whitelist_country_codes and not blacklist_country_codes %}
+                 checked="checked"
+                 {% endif %}/>
+          <label for="territories_all">All territories</label>
+          <input type="radio" name="territories" value="custom" id="territories_custom"
+                 {% if whitelist_country_codes or blacklist_country_codes %}
+                 checked="checked"
+                 {% endif %}
+                 />
+          <label for="territories_custom">Selected territories</label>
+          <div class="p-nested-inputs{% if not whitelist_country_codes and not blacklist_country_codes %} is-hidden{% endif %}">
+            <div class="js-whitelist p-multiselect__wrapper">
+              <input type="radio" name="territories_custom_type" value="whitelist" id="territories_custom_whitelist"
+                     {% if whitelist_country_codes or not blacklist_country_codes %}
+                     checked="checked"
+                     {% endif %}
+                     />
+              <label for="territories_custom_whitelist">Only these territories</label>
+              <input type="text" class="js-multiselect-input" name="whitelist_countries" value="
+                                                                                                {% for territory in whitelist_country_codes %}
```

Comment:
this part here it displayed in a strange way by github... not sure if it is really a problem.